tools/topology/sof/pipe-passthrough-playback-variable.m4

```diff
@@ -0,0 +1,46 @@
+# Low Latency Passthrough Pipeline and PCM
+#
+# Pipeline Endpoints for connection are :-
+#
+#  host PCM_P --> B0 --> sink DAI0
```

Comment:
is there actually a DAI here?

Reply:
this should have no difference to basic passthrough pipe, as this is a copy/paste/modification with only variable formats. I'll double check the existing comments what they actually read...